test/OpenTelemetry.Exporter.Geneva.Tests/OtlpProtobufMetricExporterTests.cs

```diff
@@ -0,0 +1,265 @@
+{
+    [Theory]
+    [InlineData(123)]
+    [InlineData(-123)]
+    public void LongCounterSerializationSingleMetricPoint(long value)
+    {
+        using var meter = new Meter(nameof(this.LongCounterSerializationSingleMetricPoint), "0.0.1");
+        var longCounter = meter.CreateCounter<long>("longCounter");
+        var exportedItems = new List<Metric>();
+        using var inMemoryReader = new BaseExportingMetricReader(new InMemoryExporter<Metric>(exportedItems))
+        {
+            TemporalityPreference = MetricReaderTemporalityPreference.Delta,
+        };
+
+        var resourceBuilder = ResourceBuilder.CreateDefault().Clear()
+            .AddAttributes(new[] { new KeyValuePair<string, object>("TestResourceKey", "TestResourceValue") });
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .SetResourceBuilder(resourceBuilder)
+            .AddMeter(nameof(this.LongCounterSerializationSingleMetricPoint))
+            .AddReader(inMemoryReader)
+            .Build();
+
+        longCounter.Add(value, new("tag1", "value1"), new("tag2", "value2"));
+
+        meterProvider.ForceFlush();
+
+        var buffer = new byte[65360];
+
+        var testTransport = new TestTransport();
+        var otlpProtobufSerializer = new OtlpProtobufSerializer(testTransport);
+
+        otlpProtobufSerializer.SerializeAndSendMetrics(buffer, meterProvider.GetResource(), new Batch<Metric>(exportedItems.ToArray(), exportedItems.Count));
+
+        Assert.Single(testTransport.ExportedItems);
+
+        var request = new OtlpCollector.ExportMetricsServiceRequest();
+
+        request.MergeFrom(testTransport.ExportedItems[0]);
+
+        Assert.Single(request.ResourceMetrics);
+
+        Assert.NotNull(request.ResourceMetrics[0].Resource);
+
+        AssertOtlpAttributes([new KeyValuePair<string, object>("TestResourceKey", "TestResourceValue")], request.ResourceMetrics[0].Resource.Attributes);
+
+        Assert.Single(request.ResourceMetrics[0].ScopeMetrics);
+
+        var scope = request.ResourceMetrics[0].ScopeMetrics[0];
+
+        Assert.Equal(meter.Name, scope.Scope.Name);
+
+        Assert.Single(request.ResourceMetrics[0].ScopeMetrics[0].Metrics);
+
+        var metric = request.ResourceMetrics[0].ScopeMetrics[0].Metrics[0];
+
+        Assert.Equal(longCounter.Name, metric.Name);
+
+        Assert.NotNull(metric.Sum);
+
+        Assert.Equal(1, (int)metric.Sum.AggregationTemporality);
+
+        Assert.True(metric.Sum.IsMonotonic);
+
+        Assert.Single(metric.Sum.DataPoints);
+
+        var dataPoint = metric.Sum.DataPoints[0];
+
+        Assert.Equal(value, dataPoint.AsInt);
+
+        // Assert time
+        var metricPointsEnumerator = exportedItems[0].GetMetricPoints().GetEnumerator();
+        metricPointsEnumerator.MoveNext();
+        var metricPoint = metricPointsEnumerator.Current;
+
+        Assert.Equal((ulong)TimestampHelpers.ToUnixTimeNanoseconds(metricPoint.StartTime), dataPoint.StartTimeUnixNano);
```

Comment:
nit: You can also check this without relying on the helper method in `src/`.

```suggestion
            Assert.Equal((ulong)((DateTimeOffSet)metricPoint.StartTime.ToUnixTimeSeconds() * 1000000000), dataPoint.StartTimeUnixNano);
```